src/test/java/com/zepben/cimbend/network/tracing/AssignToFeedersTest.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.cimbend.network.tracing
+
+import com.zepben.cimbend.cim.iec61970.base.core.Equipment
+import com.zepben.cimbend.cim.iec61970.base.core.Feeder
+import com.zepben.cimbend.cim.iec61970.base.core.PhaseCode
+import com.zepben.cimbend.cim.iec61970.base.core.Substation
+import com.zepben.cimbend.network.NetworkService
+import com.zepben.cimbend.testdata.TestDataCreators.*
+import com.zepben.testutils.junit.SystemLogExtension
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+import java.util.*
+import java.util.function.Consumer
+
+class AssignToFeedersTest {
+
+    @JvmField
+    @RegisterExtension
+    var systemErr: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+
+    @Test
+    fun stopsAtOpenPoints() {
+        val network = stopsAtOpenPointsNetwork()
+        Tracing.assignEquipmentContainersToFeeders().run(network)
+        val feeder = network.get(Feeder::class.java, "f")!!
+        val mRIDs: MutableList<String> = ArrayList()
+        feeder.currentEquipment.forEach(Consumer { equipment: Equipment -> mRIDs.add(equipment.mRID) })
+        MatcherAssert.assertThat<List<String>>(mRIDs, Matchers.containsInAnyOrder("fsp", "c2", "op"))
+    }
+
+    /*
+      c1       c2      c3
+     ---- fsp ---- op ----
+    */
+    private fun stopsAtOpenPointsNetwork(): NetworkService {
+        val networkService = NetworkService()
+
+        val substation = Substation().also { networkService.add(it) }
+
+        val c1 = createAcLineSegmentForConnecting(networkService, "c1", PhaseCode.A)
```

Comment:
Don't really need to have both conductors attached to the fsp as this case is tested in the other test